vagrant/hosts

```diff
@@ -23,3 +24,7 @@ ansible_dns=mesos.dev
 
 [marathon-lb]
 10.141.141.10
+
+[glusterfs_nodes]
```

Comment:
I suggest to use `_` or `-` consistently. In this case however, I think `_nodes` can be removed.